docs/docs/index.md

```diff
@@ -0,0 +1,75 @@
+---
+hide:
+- toc
+---
+![Tracee Logo >](images/tracee.png)
+
+👋 Welcome to Tracee Documentation! To help you get around, please notice the different sections at the top global menu:
+
+- You are currently in the [Getting Started](./) section where you can find general information and help with first steps.
+- In the [Tutorials](./tutorials/overview) section you can find step-by-step guides that help you accomplish specific tasks.
+- In the [Docs](./docs/overview) section you can find the complete reference documentation for all of the different features and settings that Tracee has to offer.
+- In the [Contributing](./contributing/overview) section you can find technical developer documentation and contribution guidelines.
+
+<!-- links that differ between docs and readme -->
+[installation]:./docs/install/
+[docker-guide]:./docker.md
+[kubernetes-guide]:./kubernetes.md
+[prereqs]:./docs/install/prerequisites.md
+<!-- everything below is copied from readme -->
+
+Before moving on, please consider giving us a GitHub star ⭐️. Thank you!
+
+## About Tracee
+
+Tracee is a runtime security and observability tool that helps you understand how your system and applications behave.  
+It is using [eBPF technology](https://ebpf.io/what-is-ebpf/) to tap into your system and expose that information as events that you can consume.  
+Events range from factual system activity events to sophisticated security events detect suspicious behavioral patterns.
+
+To learn more about Tracee, check out the [documentation](https://aquasecurity.github.io/tracee/). 
+
+## Quickstart
+
+To quickly try Tracee use one of the following snippets. For a more complete installation guide, check out the [Installation section][installation].  
+Tracee should run on most common Linux distributions and kernels. For more compatibility information see the [Prerequisites][prereqs] page.
+
+### Using Docker
+
+```shell
+docker run --name tracee \
+  --pid=host --cgroupns=host --privileged \
+  -v /etc/os-release:/etc/os-release-host:ro \
+  aquasec/tracee:latest
+```
+
+```shell
+docker logs tracee
+```
+
+For a complete walkthrough please see the [Docker getting started guide][docker-guide].
+
+### On Kubernetes
+
+```shell
+helm repo add aqua https://aquasecurity.github.io/helm-charts/
+helm repo update
+helm install tracee aqua/tracee --namespace tracee-system --create-namespace
+```
+
+```shell
```

Comment:
I don't think this works -- wee need to add the tracee namespace and the specific pod name to query?

Reply:
thanks for catching this